app/middlewares/sign_up_validator.js

```diff
@@ -0,0 +1,26 @@
+const errors = require('../errors');
```

Comment:
Im guessing you haven't merged the changes of the previous PRs here yet, since the name is in snake_case